calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *cnx_policy_rule_bytes / cnx_policy_rule_packets*            |
+| Explanation                      | Number of bytes or packets handled by {{prodname}} network policy rules. |
+| Threshold value recommendation   | This metric should usually be non-zero (unless expected).  A zero value indicates the rule is not matching any packets, and could be surplus to requirements. |
+| Threshold breach symptoms        | N/A                                                          |
+| Threshold breach recommendations | If this metric consistently reports a zero value over an acceptable time, you can consider removing the policy rule. |
+| Priority Level                   | Optional.                                                    |
+
+
+## Typha general metrics
+
+### Data store cache size
+
+| Data store cache size             |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *max by (instance) <(typha_cache_size>*<br />typha_cache_size[syncer="felix"]<br/>typha_cache_size[syncer="bgp"] <br/>typha_cache_size[syncer="dpi"]<br />typha_cache_size[syncer="tbd"] |
+| Explanation                      | The total number of key/value pairs in Typha's in-memory cache for all components or individual components. This metric represents the scale of the {{prodname}} data store as it tracks how many WEPs (pods and services), HEPs (hostendpoints), networksets, globalnetworksets, network policies, etc. that Typha is aware of across the entire {{prodname}} cluster.  Felix data will be the largest size and is the most critical to monitor. This is a good metric to understand how much data is in Typha. Use a query that selects the largest/Felix, monitor each of component, or sum them to get a total.<br />**Note**: If all Typhas are in sync, they should have the same value for this metric. |
+| Threshold value recommendation   | The value of this metric will depend on the scale of the Calico Federation and will always increase as WEPs, {{prodname}} network policies, and clusters are added. It is recommended to achieve a baseline first, then monitor for any unexpected increases from the baseline. |
+| Threshold breach symptoms        | Unexpected increases may indicate memory leaks and performance issues with Typha. |
+| Threshold breach recommendations | Check CPU usage on Typha pods and Kubernetes nodes. Increase resources if needed, rollout and restart Typha(s) if needed. |
+| Priority level                   | Optional.                                                    |
+
+### CPU usage
+
+| CPU usage                        |                                                              |
```

Comment:
Is there a way to merge the header cells?  A lot of the metrics tables have a very cramped top-left cell.

Reply:
Not an easy way (unless we get a plugin).